.github/workflows/build_sdk_packages.yaml

```diff
@@ -0,0 +1,33 @@
+name: SDK Packaging
```

Comment:
Working on it, not yet pushed :D I'd say auto push is ok (specially when paired with release candidates, like 1.0.0-rc.1 and ...rc.2 and then when all rounds of testing finish, tag as 1.0.0).

Reply:
@macaab26 and I discussed using a private repo for testing purposes before pushing to external instances. For info, please check https://packaging.python.org/en/latest/guides/hosting-your-own-index/

Reply:
@veekaybee here it is!!!
https://github.com/mozilla-ai/lumigator/blob/47a70ccc6f044ad4dc0d4a641640a274ee3874dc/.github/workflows/build_and_push_sdk_packages_TEST.yaml
The idea is to use a `workflow_run` trigger to avoid copying steps, but the content should be the same.

Reply:
@javiermtorres @macaab26 let's chat on setting up a private repo, that seems like overkill when we have test pypi

Reply:
After thinking a bit about it, I agree with not setting up a private test repo. Test PyPI + release candidates would seem enough safety nets for me. Even if we publish on the real PyPI we still have release candidates until a final version. Wdyt @macaab26 ?

Reply:
I'm so sorry @javiermtorres @chainlink , IDK why GitHub didn't notify me. Yeah, agree, no problem at all with Test PyPi. What I don't want (and I think we all agree on this) is having X.X.X-rcwhatever packages in the repo.

Reply:
Ok, agreed